src/main/scala/com/fulcrumgenomics/bam/EstimatePoolingFractions.scala

```diff
@@ -76,18 +79,24 @@ class EstimatePoolingFractions
   case class Locus(chrom: String, pos: Int, ref: Char, alt: Char, expectedSampleFractions: Array[Double], var observedFraction: Option[Double] = None)
 
   override def execute(): Unit = {
-    val vcfReader   = new VCFFileReader(vcf.toFile)
+    val vcfReader   = VcfSource(vcf)
     val sampleNames = pickSamplesToUse(vcfReader)
     val intervals   = loadIntervals
 
     // Get the expected fractions from the VCF
     val vcfIterator = constructVcfIterator(vcfReader, intervals, sampleNames)
-    val loci = vcfIterator.filterNot(v => this.nonAutosomes.contains(v.getContig)).map { v => Locus(
-      chrom = v.getContig,
-      pos = v.getStart,
-      ref = v.getReference.getBaseString.charAt(0),
-      alt = v.getAlternateAllele(0).getBaseString.charAt(0),
-      expectedSampleFractions = sampleNames.map { s => val gt = v.getGenotype(s); if (gt.isHomRef) 0 else if (gt.isHet) 0.5 else 1.0 }
+    val loci = vcfIterator.filterNot(v => this.nonAutosomes.contains(v.chrom)).map { v => Locus(
+      chrom = v.chrom,
+      pos   = v.pos,
+      ref   = v.alleles.ref.bases.charAt(0),
+      alt   = v.alleles.alts.head.value.charAt(0),
+      expectedSampleFractions = sampleNames.map { s =>
+        val gt = v.gt(s)
+        gt.get[IndexedSeq[Float]]("AF") match {
+          case None      => if (gt.isHomRef) 0 else if (gt.isHet) 0.5 else 1.0
```

Comment:
```suggestion
          // Developer note: it is important we only consider bi-allelic sites, since a call could be heterozygous alt1-alt2
          case None      => if (gt.isHomRef) 0 else if (gt.isHet) 0.5 else 1.0
```

Reply:
Added a different comment.  Also it makes me laugh when you start comments with "Developer note" - like what else are comments in code?  😛